composer.json

```diff
@@ -27,16 +28,17 @@
     },
     "require-dev": {
         "phpunit/phpunit":              "^5.0",
-        "zendframework/zend-diactoros": "^1.1"
+        "zendframework/zend-diactoros": "^1.1",
+        "humbug/humbug":                "dev-master"
     },
     "autoload": {
         "psr-4": {
-            "PSR7Session\\": "src/PSR7Session"
+            "PSR7Sessions\\Storageless\\": "src/Storageless"
```

Comment:
If we're trying to mimic PSR-0 here, then the path should `src/PSR7Sessions/Storageless`. Otherwise nvm :)